packages/dependency-extraction-webpack-plugin/test/fixtures/cyclic-deps/webpack.config.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * Internal dependencies
+ */
+const DependencyExtractionWebpackPlugin = require( '../../..' );
+
+module.exports = {
+	plugins: [ new DependencyExtractionWebpackPlugin() ],
```

Comment:
I think there will need to be an external module listed here that's used in one or both of the file and we can probably disable defaults as well.

Reply:
Thanks Jon! Yeah, I've added it now! 

I think I didn't realize this before: The problem that I found only affects the **externalized** modules. Do you think that's correct?